documentation/docs/60-appendix/60-glossary.md

```diff
@@ -34,9 +34,9 @@ In SvelteKit, client-side routing will be used by default, but you can skip it w
 
 ## SPA
 
-A single-page app (SPA) is an application in which all requests to the server load a single HTML file which then does client-side rendering of the requested contents based on the requested URL. All navigation is handled on the client-side in a process called client-side routing with per-page contents being updated and common layout elements remaining largely unchanged. SPAs do not provide SSR, which has the shortcoming described above. However, some applications are not greatly impacted by these shortcomings such as a complex business application behind a login where SEO would not be important and it is known that users will be accessing the application from a consistent computing environment.
+A single-page app (SPA) is an application in which all requests to the server load a single HTML file which then does client-side rendering of the requested contents based on the requested URL. All navigation is handled on the client-side in a process called client-side routing with per-page contents being updated and common layout elements remaining largely unchanged. SPA mode is highly discouraged for public-facing websites (i.e. not behind a login or only used internally at a company). It has a large performance impact by forcing two network round trips before rendering can begin. This may be acceptable if you are serving a corporate application from the local network where the network round trip is very fast, but probably is not for most websites on the internet especially when considering the latency of mobile devices.
```

Comment:
While reading this I realized that people have wildly different understandings of what an SPA is. Some call what SvelteKit does by default (SSR, then client takes over) an SPA, too. Maybe we can put a disclaimer here that there are different understandings of this matter, but this is how we're using the term throughout the site.

Reply:
Not a SvelteKit contributor, but I was following the discussion. :) I often see the term [Client Side Rendering (CSR)](https://www.patterns.dev/react/client-side-rendering/) used to refer to what you are referring to as an SPA.

Reply:
Okay, I added a bit to clarify SPA vs hybrid app